app/src/main/java/com/nmakademija/nmaakademija/fragment/ScheduleFragment.java

```diff
@@ -37,6 +40,23 @@ public static ScheduleFragment getInstance() {
         return new ScheduleFragment();
     }
 
+    @Override
+    public void onStart() {
+        super.onStart();
+        getScheduleEvents(API.nmaService);
```

Comment:
Looks strange to pass `API.nmaService`. Remove argument from `getScheduleEvents `